shared/sharedtest/util.go

```diff
@@ -5,8 +5,8 @@
 package sharedtest
 
 import (
+	"github.com/web-platform-tests/wpt.fyi/shared"
```

Comment:
Nit: move this import to the end. And while we are at it, we should use `context` instead of `golang.org/x/net/context`.